indico/modules/events/papers/lists.py

```diff
@@ -186,7 +185,7 @@ def __init__(self, event, user):
         }
         judging_unassigned_choices = {role.value: role.title for role in PaperReviewingRole
                                       if role is not PaperReviewingRole.judge}
-        self.static_items['unassigned']['filter_choices'] = OrderedDict(sorted(judging_unassigned_choices.items()))
+        self.static_items['unassigned']['filter_choices'] = judging_unassigned_choices
```

Comment:
This can be simplified into a single statement (and also sort by displayed name, not the internal enum member name):

```python
        self.static_items['unassigned']['filter_choices'] = {
            role.value: role.title
            for role in sorted(PaperReviewingRole, key=attrgetter('title'))
            if role is not PaperReviewingRole.judge
        }
```